src/components/bind-results/bindResults.html

```diff
@@ -0,0 +1,43 @@
+<div>
```

Comment:
Does this extra div do anything?  If not, I'd remove it to cut down on div-itis.